### PR TITLE
Update readme with community-plus header and security language

### PR DESCRIPTION
**Pull request by carlonewrelic**
## Proposed Release Notes
- Update readme with community-plus header and security language.
## Links
- Closes: #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,3 +74,13 @@ Issues and enhancement requests can be submitted in the [Issues tab of this repo
 Contributions are welcome (and if you submit a Enhancement Request, expect to be invited to contribute it yourself :grin:). Please review our [Contributors Guide](CONTRIBUTING.md).
 
 Keep in mind that when you submit your pull request, you'll need to sign the CLA via the click-through using CLA-Assistant. If you'd like to execute our corporate CLA, or if you have any questions, please drop us an email at opensource+newrelic-winston-logenricher-node@newrelic.com.
+
+**A note about vulnerabilities**
+
+As noted in our [security policy](/SECURITY.md), New Relic is committed to the privacy and security of our customers and their data. We believe that providing coordinated disclosure by security researchers and engaging with the security community are important means to achieve our security goals.
```
Comment (michaelgoin) [inline README.md:80]: this does not have a SECURITY.md file. we wouldn't want to point to it anyways...

recommend: https://github.com/newrelic/newrelic-winston-logenricher-node/security/policy (you may be able to put a relative link)

File: README.md [hunk 2]
```diff
@@ -72,3 +74,13 @@ Issues and enhancement requests can be submitted in the [Issues tab of this repo
...
+
+If you believe you have found a security vulnerability in this project or any of New Relic's products or websites, we welcome and greatly appreciate you reporting it to New Relic through [HackerOne](https://hackerone.com/newrelic).
+
+If you would like to contribute to this project, review [these guidelines](./CONTRIBUTING.md).
+
+To [all contributors](<LINK TO contributors>), we thank you!  Without your contribution, this project would not be what it is today.  We also host a community project page dedicated to [Project Name](<LINK TO https://opensource.newrelic.com/projects/... PAGE>).
```
Comment (michaelgoin) [inline README.md:86]: Fix the LINK TO etc... items

Comment (carlonewrelic) [reply README.md:86]: 👍  Should I update the repo to get it closer to the Open by default template?  Looks like missing a troubleshooting github issues template as well.

Comment (michaelgoin) [reply README.md:86]: There's quite a few things that need to be touched to get this up to standard. If you want to take that on (or part of that), it certainly saves us work and confusion down the road. But also, you may not have that bandwidth so up to you.

Comment (carlonewrelic) [reply README.md:86]: I'm going to go ahead and get the repo up to the standard.

**Review by michaelgoin — APPROVED** [review]

**Pull request merged**